packages/shipjs/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# [0.25.0](https://github.com/algolia/shipjs/compare/v0.24.4...v0.25.0) (2022-11-22)
+
+
+### Features
+
+* **versioning:** allow independent versioning ([#984](https://github.com/algolia/shipjs/issues/984)) ([53f2f0b](https://github.com/algolia/shipjs/commit/53f2f0b7aed881bebbc7d6f7600334e5ed990a22))
+
+
```

Comment:
### Bug Fixes

* **github:** require later version of octokit ([bb915bf](https://github.com/algolia/shipjs/commit/bb915bf617cffdc2e817e2b06719082386ea3f6a))